x-pack/plugins/maps/public/trigger_actions/visualize_geo_field_action.ts

```diff
@@ -43,6 +46,9 @@ export const visualizeGeoFieldAction = createAction<VisualizeFieldContext>({
   execute: async (context) => {
     const { app, path, state } = await getMapsLink(context);
 
+    const usageCollection = getUsageCollection();
+    usageCollection?.reportUiCounter(context.originatingApp as string, METRIC_TYPE.CLICK, `click`);
```

Comment:
The first parameter should be `APP_ID` from "maps/common/constants". Then, make the third parameter be 

```
context.originatingApp ? `visualize_geo_field_${context.originatingApp}` : 'visualize_geo_field'`
```